pkg/Microsoft.NETCore.Targets/runtime.json

```diff
@@ -10,12 +10,77 @@
                 "win10-arm-aot"
             ]
         },
+        "net45.app": {
+            "net45": [
+                "",
+                "win-x86",
+                "win-x64"
+            ]
+        },
+        "net451.app": {
+            "net451": [
+                "",
+                "win-x86",
+                "win-x64"
+            ]
+        },
+        "net452.app": {
+            "net452": [
+                "",
+                "win-x86",
+                "win-x64"
+            ]
+        },
         "net46.app": {
             "net46": [
+                "",
+                "win-x86",
+                "win-x64"
+            ]
+        },
+        "net461.app": {
+            "net461": [
+                "",
                 "win-x86",
                 "win-x64"
             ]
         },
+        "net462.app": {
+            "net462": [
+                "",
+                "win-x86",
+                "win-x64"
+            ]
+        },
+        "netcoreapp1.0.app": {
+            "netcoreapp1.0": [
+                "win7-x86",
```

Comment:
Which set is that, the RIDs or the full set of Frameworks that support contracts.  For RIDs its just the archs that we run the desktop framework on.  For RIDs for NETCoreApp1.0 its the set of concrete RIDs which we currently support in coreclr builds.